packages/jsii-rosetta/lib/rosetta-translator.ts

```diff
@@ -149,6 +149,13 @@ export class RosettaTranslator {
 function tryReadFromCache(sourceSnippet: TypeScriptSnippet, cache: LanguageTablet, fingerprinter: TypeFingerprinter) {
   const fromCache = cache.tryGetSnippet(snippetKey(sourceSnippet));
 
+  // infused snippets won't pass the full source check or the fingerprinter
```

Comment:
Why do they not pass the fingerprinter? I tried to investigate this, and I can't really make sense of it.
Verified that the fqns are the same between `fromCache.fqnsReferenced()` and `fromCache.snippet.fqnsReferenced`. The only difference is that locally I removed the `monocdk` assembly and the fingerprinter takes in all assemblies at initialization. But I fail to see how that impacts anything here.

A bit of a cop out, but I think its unnecessary. We _cant_ translate infused examples, so we might as well take what we've got. Thoughts?

Reply:
I'm surprised as well, but I agree with the solution. It's fine like this.